package-coverage/main.go

```diff
@@ -41,6 +42,7 @@ func main() {
 	flag.BoolVar(&print, "p", false, "print coverage to stdout")
 	flag.BoolVar(&slack, "slack", false, "output coverage to slack")
 	flag.StringVar(&ignoreDirs, "i", `./\.git.*|./_.*`, "ignore regex specified directory")
+	flag.StringVar(&ignoreFiles, "j", `mock.*\.go`, "ignore regex files")
```

Comment:
I dont like the default value.  It's rather specific to your use-case and actually not how I would personally prefer to use it